plotting-app/hid-driver/src/main/java/org/unknown/plottingapp/hiddriver/models/HIDState.java

```diff
@@ -0,0 +1,63 @@
+package org.unknown.plottingapp.hiddriver.models;
+
+public class HIDState {
+    private StickPosition leftLR;
+    private StickPosition leftUD;
+    private StickPosition rightLR;
+    private StickPosition rightUD;
+
+    public HIDState() {
+        leftLR = StickPosition.CENTER;
+        leftUD = StickPosition.CENTER;
+        rightLR = StickPosition.CENTER;
+        rightUD = StickPosition.CENTER;
+    }
+
+    public void setState(int rightX, int rightY, int leftX, int leftY) {
+        rightLR = getHorizontal(rightY);
+        rightUD = getVertical(rightX);
+        leftLR = getHorizontal(leftY);
+        leftUD = getVertical(leftX);
+    }
+
+    public StickPosition getLeftLR() {
+        return leftLR;
+    }
+
+    public StickPosition getLeftUD() {
+        return leftUD;
+    }
+
+    public StickPosition getRightLR() {
+        return rightLR;
+    }
+
+    public StickPosition getRightUD() {
+        return rightUD;
+    }
+
+    @Override
+    public String toString() {
+        return "HIDState{" +
+                "leftLR=" + leftLR +
+                ", leftUD=" + leftUD +
+                ", rightLR=" + rightLR +
+                ", rightUD=" + rightUD +
+                '}';
+    }
+
+
+    private StickPosition getHorizontal(int yPos) {
+        if (yPos > 700) return StickPosition.RIGHT;
```

Comment:
how did we come up with these number? (700, 200, ..)

Reply:
The device is an analog device that uses an ADC to map signals to a range of [0,1024)